app/view/toolbar/MapFooter.js

```diff
@@ -21,6 +21,16 @@ Ext.define('CpsiMapview.view.toolbar.MapFooter', {
             xtype: 'basigx-combo-scale'
         }, {
             xtype: 'basigx-panel-coordinatemouseposition',
+            showMarker: true,
+            markerStyle: new ol.style.Style({
```

Comment:
Meanwhile there are quite many predefined styles defined in the style util: https://github.com/compassinformatics/cpsi-mapview/blob/master/app/util/Style.js

Just wondering if it would make sense to store this style in the style util as well.

Reply:
Good point! Will update accordingly